db/schema_tables.cc

```diff
@@ -1774,7 +1780,11 @@ static shared_ptr<cql3::functions::user_aggregate> create_aggregate(replica::dat
 
     bytes_opt initcond = std::nullopt;
     if (initcond_str) {
-        initcond = state_type->from_string(initcond_str.value());
+        auto expr = cql3::util::do_with_parser(*initcond_str, std::mem_fn(&cql3_parser::CqlParser::term));
```

Comment:
There is some types that are not compatible with the previous format. They couldn't actually have been avoided, because the previous format couldn't be properly parsed

Reply:
Please explain the cases and problems in the changelog.

Reply:
I meant examples.

Reply:
(of where we lost compatibility)

Reply:
I've added 2 examples in the last rebase